worker/src/durable.rs

```diff
@@ -495,6 +564,46 @@ impl<'a> ListOptions<'a> {
     }
 }
 
+pub struct ScheduledTime {
+    date: js_sys::Date,
+}
+
+impl ScheduledTime {
+    pub fn new(date: js_sys::Date) -> Self {
+        Self { date }
+    }
+}
+
+impl From<i64> for ScheduledTime {
+    fn from(timestamp: i64) -> Self {
+        ScheduledTime {
+            date: js_sys::Date::new(&Number::from(timestamp as f64)),
+        }
+    }
+}
+
+impl From<DateTime<Utc>> for ScheduledTime {
+    fn from(date: DateTime<Utc>) -> Self {
+        ScheduledTime {
+            date: js_sys::Date::new(&Number::from(date.timestamp_millis() as f64)),
+        }
+    }
+}
+
+#[derive(Default, Serialize)]
+pub struct GetAlarmOptions {
```

Comment:
These structs their fields marked as `pub` so they be set by anyone and the structs should also derive `Debug` and `Clone`.